stand/efi/loader/Makefile

```diff
@@ -30,6 +30,65 @@ SRCS=	autoload.c \
 	gfx_fb.c \
 	8x16.c
 
+# ACPICA Loader Initialization
+SRCS+= acpi_detect.c
+
+.if ${MACHINE_ARCH} == "amd64"
+SRCS+= OsdMemory.c osunixxf.c
```

Comment:
So for all these files, I'd be tempted to create a stand/efi/libacpi and build them all and put them all in there. Since they are all listed here, they all build three times. It's fine for development, but it's a lot to build three times. Not an immediate thing, but before commit. Fortunately, moving it a library is pretty straight forward (though I've done it so many times I may be forgetting a gotcha).